### ISSUE TITLE
Semantic tags

### ISSUE THREAD

**Issue by Kyzyl-ool** [gravity-ui/page-constructor#483]
Need to use semantic tags correctly in the project. Here are the suggested changes:

1. Need to have an opportunity to change the title level [here](https://github.com/gravity-ui/blog-constructor/blob/a95c55bd1ff98f86a110f33ac3275b590ee5db8e/src/components/PostCard/PostCard.tsx#L74) and [here](https://github.com/gravity-ui/page-constructor/blob/a4d4129af7d1b82e149ee6ced215d5733ad1b2f7/src/utils/blocks.ts#L43)

1. Should use `h3` tag instead of `h4` for `pc-title-item__text` [here](https://preview.yandexcloud.dev/page-constructor/?path=/story/blocks-tabs--default)

1. Use `h3` tag instead of `h4` [here](https://github.com/gravity-ui/page-constructor/blob/e0813b32f61ae4d2526c461560a78fcfb460c271/src/blocks/Questions/Questions.tsx#L69)

1. `ExtendedFeaturesBlock`: as heading tag in header element is `h2`, the next used heading tag should be `h3` to meet the accessibility requirements. If heading tag of entire block is not used, each item should use `h2` tag instead of `h3`.

**Comment by Kyzyl-ool** [comment]
The appropriate PR was created https://github.com/gravity-ui/blog-constructor/pull/103
